packages/web/src/components/ui/CardList/CardList.tsx

```diff
@@ -1,19 +1,14 @@
 import React from 'react';
 import styles from './CardList.module.scss';
-import Card, { CardProps } from '../Card/Card';
 import clsx from 'clsx';
 
 type Props = {
-  cards: CardProps[];
-  variant?: 'default' | 'vertical';
+  className?: string;
+  children: React.ReactNode;
 };
 
-export default function CardList({ cards, variant = 'default' }: Props) {
-  return (
-    <div className={clsx(styles.grid, styles[variant])}>
-      {cards.map((card) => (
-        <Card key={card.id} {...card} />
-      ))}
-    </div>
-  );
-}
+const CardList = ({ className, children }: Props) => (
+  <div className={clsx(styles.grid, className)}>{children}</div>
```

Comment:
nit: Generally, a `className` prop is a sign of bad component design. Component styles should be scoped to its own only. We'll probably revisit this later as I'm not sure on all of the use cases here.